### DISCUSSION TITLE
How to Disable Vim/nvim mode Notifications in Noice.nvim

### DISCUSSION THREAD

**Discussion by sharjeelmazhar** [folke/noice.nvim#965] [General]
Hi there!

I’m using Noice.nvim in my Neovim setup and find the notifications about entering insert and visual modes quite distracting. I would like to know if there’s a way to suppress these specific notifications without disabling all notifications from Noice.

I've tried looking into the documentation and existing discussions, but I haven't found a straightforward solution. Any guidance or code snippets to help me achieve this would be greatly appreciated!

Thanks in advance for your help!

**Comment by ghaarm** [comment]
Hi,

I think I just found the solution here https://github.com/folke/noice.nvim/issues/425#issuecomment-1605759645 

added the following to my noice.lua and it works.

  vim.opt.showmode = false

like:
    config = function()
      vim.opt.showmode = false

Hope it works for you.
Cheers
